src/NUnitEngine/nunit.engine/Services/InProcessTestRunnerFactory.cs

```diff
@@ -52,13 +52,21 @@ public virtual ITestEngineRunner MakeTestRunner(TestPackage package)
 
             switch (domainUsage)
             {
+                default:
```

Comment:
Well, you learn something every day. It is such a convention that the default always goes at the end of a switch that I actually went out and read that section of the C# spec to see if it 'just works' or if it is in the spec. A literal read of the spec says it is fine.